---
title: InvalidBaseImagePlatform
description: Base image platform does not match expected target platform
aliases:
  - /go/dockerfile/rule/invalid-base-image-platform/
---

## Output

```text
```

## Description

## Examples

❌ Bad:

```dockerfile
```

✅ Good:

```dockerfile
```

