---
title: UndeclaredArgInFrom
description: FROM command must use declared ARGs
aliases:
  - /go/dockerfile/rule/undeclared-arg-in-from/
---

## Output

```text
```

## Description

## Examples

❌ Bad

```dockerfile
```

✅ Good

```dockerfile
```

